docs/installation.rst

```diff
@@ -20,6 +20,14 @@ Once that's done, you should add ``channels`` to your
 That's it! Once enabled, ``channels`` will integrate itself into Django and
 take control of the ``runserver`` command. See :doc:`getting-started` for more.
 
+.. note::
+  Please be wary of any other app that require an overloaded or
+  replacement ``runserver`` command. Channels provides a separate
+  ``runserver`` command and may conflict with it. An example
+  of such a conflict is with `whitenoise.runserver_nostatic <https://github.com/evansd/whitenoise/issues/77>`
+  from the `whitenoise <https://github.com/evansd/whitenoise>` app.
```

Comment:
Could you add a paragraph about how to resolve the issue (either moving `channels` to the top of `INSTALLED_APPS`, or removing the offending app?)